### PR TITLE
Add missing params for loopback_sim

### PR DESCRIPTION
**Pull request by adivardi**

### CODE REVIEW

File: configuration/packages/configuring-loopback-sim.rst [hunk 1]
```diff
@@ -61,13 +61,57 @@ Parameters
...
   Description
-    The scan frame to use to publish a scan for collision monitor's happiness
+    The scan frame to use to publish a scan
+
+:enable_stamped_cmd_vel:
+
+  ============== ==============
+  Type           Default
+  -------------- --------------
+  string         `false`
```
Comment (SteveMacenski) [inline configuration/packages/configuring-loopback-sim.rst:77]: Remove the "`" to be inline with other param tables. 

Also, for rst format, you would have had to use 2x of those to get it to consider a code block. 1x is in italics

**Review by SteveMacenski — CHANGES_REQUESTED** [review]
Otherwise, happy to merge

File: configuration/packages/configuring-loopback-sim.rst (hunk 1 above)
Comment (adivardi) [reply configuration/packages/configuring-loopback-sim.rst]: fixed both issues

**Review by SteveMacenski — APPROVED** [review]
Sorry for the delay, we had ROSCon, ROS-I, and then I took a short vacation while in Europe. I'm getting back to work this morning :-)

**Pull request merged**